Software/a314fs/a314fs.py

```diff
@@ -688,6 +690,14 @@ def process_same_lock(key1, key2):
     else:
         return struct.pack('>HH', 0, LOCK_SAME_VOLUME)
 
+def process_disk_size_info():
+    logger.debug('ACTION_DISK_SIZE_INFO')
+    total, used, free = shutil.disk_usage(SHARED_DIRECTORY)
+    total_blocks = total // 512
+    used_blocks = used // 512
+    block_size = 512
```

Comment:
Could there be any benefit of using the actual block size of the pi filesystem instead of claiming to use 512 bytes?

Reply:
In the latest commit there is a constant INFO_BLOCK_SIZE in the Python code, that could later be changed in some smarter way.